### PR TITLE
Correctly generate doc elements for bitfields

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: src/codegen/doc/mod.rs [hunk 1]
```diff
@@ -79,9 +79,9 @@ macro_rules! impl_function_like_type {
     };
 }
 
-impl_to_stripper_type!(Member, Variant);
+impl_to_stripper_type!(Member, Const);
 impl_to_stripper_type!(Enumeration, Enum);
-impl_to_stripper_type!(Bitfield, Type);
+impl_to_stripper_type!(Bitfield, Struct);
```
Comment (EPashkin) [inline src/codegen/doc/mod.rs:84]: Just to be sure: This change really needed?

Comment (GuillaumeGomez) [reply src/codegen/doc/mod.rs:84]: Yes, they are declared as struct and need to be stored as such, otherwise rustdoc-stripper doesn't "recognize" them.

**Comment by GuillaumeGomez** [comment]
@EPashkin By the way, any idea what's going on with the windows builds?

**Comment by EPashkin** [comment]
Seems libsecret is not installed, I don't have time to look more

**Comment by EPashkin** [comment]
@GuillaumeGomez Thanks

**Pull request merged**